src/move_basic.cpp

```diff
@@ -465,19 +476,27 @@ void MoveBasic::executeAction(const move_base_msgs::MoveBaseGoalConstPtr& msg)
          }
     }
     else {
-         drivingFrame = preferredDrivingFrame;
+      drivingFrame = preferredDrivingFrame;
     }
 
     if (!transformPose(frameId, drivingFrame, goal, goalInDriving)) {
-         abortGoal("MoveBasic: Cannot determine robot pose in driving frame");
+         abortGoal("MoveBasic: Cannot determine goal pose in driving frame");
          return;
     }
 
     tf2::Transform goalInBase = currentDrivingBase * goalInDriving;
+    {
+       double x, y, yaw;
+       getPose(goalInBase, x, y, yaw);
+       ROS_INFO("MoveBasic: Goal in %s  %f %f %f", baseFrame.c_str(),
+             x, y, rad2deg(yaw));
+    }
 
     tf2::Vector3 linear = goalInBase.getOrigin();
+    // Don't use linear.length() because z is not relevant
+    double dist = sqrt(linear.x() * linear.x() + linear.y() * linear.y());
```

Comment:
`sqrt` should be `std::sqrt` or we could put a `using std:sqrt` at the top of the file, which is less evil than `using namespace std`

Reply:
Only doing the distance computation once, and having it ignore Z is a good plan :+1: